syntax = "proto3";

package trainings;

message Training {
  int32 id = 1;
  string name = 2;
  string description = 3;
}

message GetTrainingById {
  int32 id = 1;
}

service Trainings {
   rpc GetTraining(GetTrainingById) returns (Training) {}

   rpc SaveTraining(Training) returns (Training) {}
}